wireguard4helm/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common debconf-utils iptables curl && \
+    add-apt-repository --yes ppa:wireguard/wireguard && \
+    apt-get update && \
+    echo resolvconf resolvconf/linkify-resolvconf boolean false | debconf-set-selections && \
+    apt-get install -y iproute2 wireguard-dkms wireguard-tools curl resolvconf
+
+COPY startup.sh /.
+
+ENTRYPOINT ["/startup.sh"]
```

Comment:
@iMartyn let's use the convention of entrypoint binaries: `docker-entrypoint.sh` or `entrypoint.sh`

Reply:
sure, if that's the convention you want.  I based on the jess image, an iteration later would be not to use a full ubuntu image.